src/system/web-setup-extension-manager.md

```diff
@@ -58,7 +61,9 @@ _Installed Extensions_
 1. Follow the onscreen instructions to complete the following:
 
     - Step 1: Readiness Check
+
     - Step 2: Create Backup
+
     - Step 3: Component Install
 
 ### Step 1: Readiness Check
```

Comment:
All H2, H3, and up should be Sentence case/capitalization

```suggestion
### Step 1: Readiness check
```

Reply:
I removed this bullet summary because it was redundant.